buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -25,11 +23,14 @@ object Versions {
             const val snakeYml = "2.3"
             const val jetBrainsAnnotations = "24.1.0"
             const val junit = "5.11.0"
+            const val nbt = "6.1"
         }
     }
     
     object Fabric {
-        const val fabricAPI = "0.104.0+${Mod.minecraft}"
+//        const val fabricAPI = "0.104.0+${Mod.minecraft}"
+        const val fabricAPI = "0.105.4+1.21.2"
+        const val cloud = "2.0.0-beta.9"
```

Comment:
Why was cloud split between fabric & bukkit?

Reply:
because cloud fabric and cloud bukkit are two separate packages which can have independent version numbers

Reply:
ah I see